File: Tipper/app/src/main/java/com/example/tipper/Question.java
package com.example.tipper;

import java.util.ArrayList;
import java.util.List;

public class Question {
    private String question;
    private List<String> answers;
    private String correctAnswer;

    public Question(String correctAnswer, String question) {
        this.answers = new ArrayList<>();
        this.correctAnswer = correctAnswer;
        this.question = question;
    }

    public List<String> getAnswers() {
        return answers;
    }

    public void setAnswers(List<String> answers) {
        this.answers = answers;
    }

    public String getCorrectAnswer() {
        return correctAnswer;
    }

    public void setCorrectAnswer(String correctAnswer) {
        this.correctAnswer = correctAnswer;
    }

    public String getQuestion() {
        return question;
    }

    public void setQuestion(String question) {
        this.question = question;
    }

    public void addAnswer(String answer){
        this.answers.add(answer);
    }
}

File: kotlin/app/src/main/java/com/example/tipper/Question.kt
package com.example.tipper

import java.util.ArrayList

class Question(correctAnswer: String, question: String) {
    var question: String
    private var answers: MutableList<String>
    var correctAnswer: String
    fun getAnswers(): List<String> {
        return answers
    }

    fun setAnswers(answers: MutableList<String>) {
        this.answers = answers
    }

    fun addAnswer(answer: String) {
        answers.add(answer)
    }

    init {
        answers = ArrayList()
        this.correctAnswer = correctAnswer
        this.question = question
    }
}